cinema/serializers.py

```diff
@@ -94,5 +101,6 @@ class Meta:
 
 
 class MovieSessionDetailSerializer(MovieSessionSerializer):
+
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
```

Comment:
remove many parameter, it's False by default

Reply:
> remove many parameter, it's False by default

Oh sorry, some mentors told that it's imortant to writedown it, but I ll change